Release talent 0.3.0 (#8769)
googleapis/google-cloud-python: talent/CHANGELOG.md

[1]: https://pypi.org/project/google-cloud-talent/#history

## 0.3.0

07-24-2019 17:36 PDT

### Implementation Changes

> - search_profiles returns iterable of SummarizedProfile instead of HistogramQueryResult (via synth). ([#7962](https://github.com/googleapis/google-cloud-python/pull/7962))

Reviewer: ```suggestion
- Return iterable of `SummarizedProfile` from `search_profiles` rather than`HistogramQueryResult` (via synth). ([#7962](https://github.com/googleapis/google-cloud-python/pull/7962))
```
